service/cloudconfigv4/cloud_config.go

```diff
@@ -0,0 +1,112 @@
+package cloudconfigv4
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/certs"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_2_0_0"
```

Comment:
I think what you actually want to do here is using `v_2_0_1`, because this is the patch you created for the templates or is this here the copy PR? Then again, IMO the copy PR should not be more than `cp src dst`. I am as the reviewer do not know what this PR is about now and how it should be.

Reply:
Ah, missed main part with `v_2_0_1`. Regarding `cp src dst` it just does not compile after copy paste. For now added more description for PR.

Reply:
When you only copy you add a new package which is not imported by any code. It does compile then. Maybe describe what you did in detail and show what exactly caused it to not compile. When you follow the suggested workflow you will not have any compile issues.